go.mod

```diff
@@ -3,14 +3,16 @@ module github.com/tinkerbell/pbnj
 go 1.15
 
 require (
-	github.com/bmc-toolbox/bmclib v0.4.2
+	github.com/bmc-toolbox/bmclib v0.4.4
+	github.com/coreos/go-etcd v2.0.0+incompatible // indirect
```

Comment:
`go mod tidy` needed here? or are we using etcd some where?

Reply:
ah good point. will do